xclim/indices/_conversion.py

```diff
@@ -1180,3 +1181,574 @@ def potential_evapotranspiration(
+                * tdb
+                * delta_t_tr
+                * delta_t_tr
+                * pa
+                * pa
+                * pa
+                + (1.73825715 * (10 ** (-6)))
+                * v
+                * delta_t_tr
+                * delta_t_tr
+                * pa
+                * pa
+                * pa
+                + (-4.09087898 * (10 ** (-7)))
+                * delta_t_tr
+                * delta_t_tr
+                * delta_t_tr
+                * pa
+                * pa
+                * pa
+                + 0.614155345 * pa * pa * pa * pa
+                + (-0.0616755931) * tdb * pa * pa * pa * pa
+                + 0.00133374846 * tdb * tdb * pa * pa * pa * pa
+                + 0.00355375387 * v * pa * pa * pa * pa
+                + (-5.13027851 * (10 ** (-4))) * tdb * v * pa * pa * pa * pa
+                + (1.02449757 * (10 ** (-4))) * v * v * pa * pa * pa * pa
+                + (-0.00148526421) * delta_t_tr * pa * pa * pa * pa
+                + (-4.11469183 * (10 ** (-5))) * tdb * delta_t_tr * pa * pa * pa * pa
+                + (-6.80434415 * (10 ** (-6))) * v * delta_t_tr * pa * pa * pa * pa
+                + (-9.77675906 * (10 ** (-6)))
+                * delta_t_tr
+                * delta_t_tr
+                * pa
+                * pa
+                * pa
+                * pa
+                + 0.0882773108 * pa * pa * pa * pa * pa
+                + (-0.00301859306) * tdb * pa * pa * pa * pa * pa
+                + 0.00104452989 * v * pa * pa * pa * pa * pa
+                + (2.47090539 * (10 ** (-4))) * delta_t_tr * pa * pa * pa * pa * pa
+                + 0.00148348065 * pa * pa * pa * pa * pa * pa
+            )
+
+        def exponential(tas):
+            g = [
+                -2836.5744,
+                -6028.076559,
+                19.54263612,
+                -0.02737830188,
+                0.000016261698,
+                (7.0229056 * np.power(10.0, -10)),
+                (-1.8680009 * np.power(10.0, -13)),
+            ]
+            tk = tas + 273.15
+            es = 2.7150305 * np.log1p(tk)
+            for count, i in enumerate(g):
+                es = es + (i * np.power(tk, count - 2))
+            es = np.exp(es) * 0.01
+            return es
+
+        tas = np.array(tas)
+        tr = np.array(tr)
+        sfcWind = np.array(sfcWind)
+        hurs = np.array(hurs)
+
+        eh_pa = exponential(tas) * (hurs / 100.0)
+        delta = tr - tas
+        pa = eh_pa / 10.0
+
+        utci_approx = optimized(tas, sfcWind, delta, pa)
+
+        tas_valid = valid_range(tas, (-50.0, 50.0))
+        tr_valid = valid_range(tr - tas, (-30.0, 30.0))
+        sfcWind_valid = valid_range(sfcWind, (0.5, 17.0))
+        valid = ~(np.isnan(tas_valid) | np.isnan(tr_valid) | np.isnan(sfcWind_valid))
+        utci_approx = np.where(valid, utci_approx, np.nan)
+
+        return np.round_(utci_approx, 1)
+
+    tr = tas.copy()
```

Comment:
Only if `tr is None`!

Also, I think you're missing a conversion of `sfcWind`.